lib/include/jxl/decode.h

```diff
@@ -877,7 +877,8 @@ JXL_EXPORT JxlDecoderStatus JxlDecoderSetOutputColorProfile(
  * @param cms structure representing a CMS implementation. See @ref
  * JxlCmsInterface for more details.
  */
-JXL_EXPORT void JxlDecoderSetCms(JxlDecoder* dec, JxlCmsInterface cms);
```

Comment:
What's the reason for this change?

What are the lifetime requirements for the cms parameter?

Reply:
I think this was due to some problem when cms-refactoring. 
Perhaps let's change it back, so we don't have to worry about the lifetime of `cms`. 
Plan to do something like this:
https://github.com/mo271/libjxl/compare/6a61ca734d29479b23aafeb208f6968ddc03988c..33984446603d920a9350f60d82bc91f65fb19010

Reply:
reverted